cmd/kubeconfig/create.go

```diff
@@ -23,6 +23,7 @@ func kubeconfigCreateCmd() *cobra.Command {
 	var (
 		groups                  string
 		certificateExpiresAfter time.Duration
+		contextName             string
```

Comment:
Are there any constraints on the context name that need to be checked, such as length or forbidden characters? At the very least, we should error out on an empty context name.

Reply:
No forbidden characters. I’ve added validation to require at least one character (same as username), so an empty context name now errors.